well, I can only describe what I am seeing here and that is, from a PowerShell and from a cmd prompt:

npm start:

```
> no-skeleton.docpad@0.1.0 start C:\temp\promise-nuggets
> node_modules/docpad/bin/docpad-server

'node_modules' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
npm ERR! node v5.5.0
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE
npm ERR! no-skeleton.docpad@0.1.0 start: `node_modules/docpad/bin/docpad-server`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the no-skeleton.docpad@0.1.0 start script 'node_modules/docpad/bin/docpad-server'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the no-skeleton.docpad package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node_modules/docpad/bin/docpad-server
npm ERR! You can get their info via:
npm ERR!     npm owner ls no-skeleton.docpad
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\temp\promise-nuggets\npm-debug.log
```

and for docpad-server:
'docpad-server' is not recognized as an internal or external command,
operable program or batch file.

But no matter, I can assume package.json ignored :)

--- a/package.json
+++ b/package.json
@@ -14,8 +14,8 @@
     "docpad-plugin-eco": "*",
     "docpad-plugin-marked": "*"
   },
-  "main": "node_modules/docpad/bin/docpad-server",
+  "main": "docpad-server",
   "scripts": {
-    "start": "node_modules/docpad/bin/docpad-server"
+    "start": "docpad-server"
   }
 }

